pkg/validation/app.go

```diff
@@ -19,6 +20,7 @@ const (
 	catalogNotFoundTemplate         = "catalog %#q not found"
 	nameTooLongTemplate             = "name %#q is %d chars and exceeds max length of %d chars"
 	nameNotFoundReasonTemplate      = "name is not specified for %s"
+	namespaceNotAllowed             = "namespace %s is not allowed"
```

Comment:
```suggestion
	targetNamespaceNotAllowed             = "target namespace %s is not allowed"
```

I like the name of the method and I think we should use target here too. As otherwise its too easy to mix it with the namespace of the app CR.

Reply:
Makes sense!